src/main/java/de/tum/in/www1/artemis/repository/tutorialgroups/TutorialGroupSessionRepository.java

```diff
@@ -22,10 +22,10 @@
 public interface TutorialGroupSessionRepository extends JpaRepository<TutorialGroupSession, Long> {
 
     @Query("""
-            SELECT tutorialGroupSession
-            FROM TutorialGroupSession tutorialGroupSession
-            WHERE tutorialGroupSession.tutorialGroup.id = :#{#tutorialGroupId}
-            AND tutorialGroupSession.status = :#{#status}
+                SELECT tutorialGroupSession
+                FROM TutorialGroupSession tutorialGroupSession
+                WHERE tutorialGroupSession.tutorialGroup.id = :#{#tutorialGroupId}
+                AND tutorialGroupSession.status = :#{#status}
```

Comment:
Was this by accident?

Reply:
Yes! Reverted it in c3df8045eebba81cf39eb29507abed207ba77cc9